### ISSUE TITLE
Links to documentation broken

### ISSUE THREAD

**Issue by stuaxo** [bconstantin/django_polymorphic#28]
These should be moved to github (if available) and the links fixed.

In particular I'd like to see Quickstart as this project looks really useful :/

**Comment by stuaxo** [comment]
Oops just noticed this is a dupe of  https://github.com/chrisglass/django_polymorphic/issues/8  so maybe just the links need fixing.

**Comment by vdboor** [comment]
Yes, this repository is stale, as the original author has disappeared from the Internet.
You can report new issues at https://github.com/chrisglass/django_polymorphic, see my note at #31
